jest.config.ts

```diff
@@ -276,8 +278,9 @@ const config: Config.InitialOptions = {
       init: {
         // jest project under Sentry organization (dev productivity team)
         dsn: 'https://3fe1dce93e3a4267979ebad67f3de327@sentry.io/4857230',
-        environment: CI ? 'ci' : 'local',
-        tracesSampleRate: 1,
+        // Use production env to reduce sampling of commits on master
+        environment: CI ? (IS_MASTER_BRANCH ? 'ci:master' : 'ci:pull_request') : 'local',
+        tracesSampleRate: CI ? 1 : 0.5,
```

Comment:
Do we want to change sample rate by `CI` or `IS_MASTER_BRANCH`?

Reply:
attempting to prioritize CI for now by sending fewer in dev/local. but i might do that too